### PR TITLE
Clip on door frame

### PR DESCRIPTION
**Pull request by scanlory**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
This mod adds a frame around the front door panels to stiffen and seal the doors when using modified hinges.

#### How was it tested? 
The modification was printed and tested on my V2.4.

### CODE REVIEW

File: printer_mods/scanlory/clip_on_door_frame/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# V2.4 Clip-On Door Frames #
+
+ This design builds on [Alexander-T-Moss](https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/AlexanderT-Moss/270-Clamping-Hinges) 270 degree hinges by adding a clip-on frame around the front doors to provide better sealing and stiffining of the front door panels.
```
Comment (cbc02009) [inline printer_mods/scanlory/clip_on_door_frame/README.md:3]: please use a relative link here.

File: printer_mods/scanlory/clip_on_door_frame/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ ## Assembly ##
+
+ 1. Attach one angle_frame_1 to the bottom of the left door and one angle_frame_1 to the top of the right door.
+ 2. Attach one angle_frame_2 to the top of the left door and one angle_frame_2 to the bottom of the right door.
+ 3. If you have three door hinges installed, install two straight_frame_short sections to each door.  If you have two door hinges installed, install one straight_frame_long section to each door.
+ 4. If you are using the stock V2.4 magnetic door latch, install the magnets into the extended_door_latch and replace the existing door latches with the extended version.
+ 
+  ## Installed On My 2.4 ##
+
+ ![full_front_view.jpg](https://github.com/scanlory/VoronUsers/blob/clip_on_door_frame/printer_mods/scanlory/clip_on_door_frame/Images/full_front_view.jpg)
```
Comment (cbc02009) [inline printer_mods/scanlory/clip_on_door_frame/README.md:50]: please change to relative links.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/scanlory/clip_on_door_frame/README.md (hunk 2 above)
Comment (scanlory) [reply printer_mods/scanlory/clip_on_door_frame/README.md]: Updated to relative links.

**Review by cbc02009 — APPROVED** [review]

File: printer_mods/scanlory/clip_on_door_frame/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# V2.4 Clip-On Door Frames #
+
+ This design builds on [Alexander-T-Moss](../AlexanderT-Moss/270-Clamping-Hinges) 270 degree hinges by adding a clip-on frame around the front doors to provide better sealing and stiffining of the front door panels.
```
Comment (cbc02009) [inline printer_mods/scanlory/clip_on_door_frame/README.md:3]: Needs one more ../ for the link to work.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**